code-challenges.test.js

```diff
@@ -16,35 +16,107 @@
 // a) Create a test with an expect statement using the variable provided. 
 // HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
+//creating the test removeAndShuffle and describe what the test will do
+//looked at the resource given, use the method arrayContaining
+
+describe("removeAndShuffle", () => {
+
 const colors1 = ["purple", "blue", "green", "yellow", "pink"]
-// Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
-// Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+    // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
+const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"] 
+    // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+
+it("that takes in an array, removes the first item from the array and shuffles the remaining content", () => {
+    expect(removeAndShuffle(colors1)).toEqual(expect.arrayContaining(["yellow", "blue", "pink", "green"]))
+    expect(removeAndShuffle(colors2)).toEqual(expect.arrayContaining(["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]))
+})
+})
 
+//output: ReferenceError: removeAndShuffle is not defined, amazing fail
 
 // b) Create the function that makes the test pass.
 
+//creating a function with the name of removeAndShuffle, in this function I want it to remove the first value inside of the array, then have a new array with the left over words scrambled inside of a new array
+//inside the function I will be using the method .slice(1) in order to remove the first value (easy part)
+//now hard part I want to make the new array that scrambles the values in a random order, iterate through array with for loop, use math.random to randomize the order 
+
+
+const removeAndShuffle = (array) => {
```

Comment:
Nice work on this problem!  If you want to stretch yourself, consider how you could use a higher-order function instead of for loops.